utils/sim_configuration/generateSqn.ts

```diff
@@ -0,0 +1,18 @@
+import * as crypto from "crypto";
+
+export function generateSqn(): string {
+    // 48 bytes = 6 bytes (each byte is 8 bits)
```

Comment:
```suggestion
    // 48 bits = 6 bytes (each byte is 8 bits)
```
but i think that you can remove most of the comments

Reply:
The comment is  fixed and some comments are removed.